hash/pc98_cd.xml

```diff
@@ -2649,6 +2890,26 @@ license:CC0
 		</part>
 	</software>
 
+	<!-- PC-9801 / FM Towns hybrid -->
+	<software name="if">
+		<!--
+		Origin: redump.org
+		<rom name="if - Invitations from Fantastic Stories (Japan).bin" size="10231200" crc="dada6d70" sha1="5b2b80e25fc07cb805ffb8219e7761a1bebeb47c"/>
+		<rom name="if - Invitations from Fantastic Stories (Japan).cue" size="113" crc="018a1c52" sha1="4975b59268879cacae2b196120d4deabe8287a64"/>
+		-->
+		<description>if - Invitations from Fantastic Stories</description>
+		<year>1993</year>
+		<publisher>アクティブ (Active)</publisher>
+		<info name="serial" value="HME-152 / MTC-1051"/>
+		<info name="alt_title" value="イフ" />
```

Comment:
The cover art appears to show the katakana above the English subtitle – should it be included in the `alt_title` as well? https://vndb.org/v8724